src/app/core/services/attachment/attachment.service.ts

```diff
@@ -24,9 +24,9 @@ export class AttachmentService {
     ) {
     }
 
-    getActionSheetButtons(type): any {
+    getActionSheetButtons(profileImageData): any {
         let buttons = [];
-        this.utils.getActionSheetButtons(type).forEach(element => {
+        this.utils.getActionSheetButtons(profileImageData).forEach(element => {
```

Comment:
here u have to pass type right? not image data completely

Reply:
Remove current photo option check whole profile image data is needed